src/api/polkadot/connectApi.ts

```diff
@@ -44,7 +43,7 @@ const loadAccounts = async (api: ApiPromise) => {
     {
       genesisHash: api.genesisHash,
       isDevelopment,
-      type: 'ed25519',
+      ss58Format: 5,
```

Comment:
the token format is different from chain to chain (Dusty, Substrate dev node, and Plasm Main net have a different ss58 prefix). So please make this value dynamically parsed from the chain itself.